src/lib/hooks/useAcceptBid.tsx

```diff
@@ -0,0 +1,49 @@
+import { useZnsContracts } from 'lib/contracts';
+import { tryFunction } from 'lib/utils';
+import { ethers } from 'ethers';
+
+export interface AcceptBidParams {
+	signature: ethers.BytesLike;
+	auctionId: number;
+	bidder: string;
+	bid: number;
+	nftAddress: string;
+	tokenId: number;
+	minBid: number;
+	startBlock: number;
+	expireBlock: number;
+}
+
+interface StakingControllerHooks {
+	acceptBid: (params: AcceptBidParams) => Promise<ethers.ContractTransaction>;
+}
+
+export function useAcceptBid(): StakingControllerHooks {
```

Comment:
Lets call this `useZAuction` or something like that since we extend this in the future